app/jobs/application_job.rb

```diff
@@ -13,10 +13,11 @@ def symphony_client
       @symphony_client || SymphonyClient.new
     end
 
-    def badge(message: message)
+    def badge(message: message, badge_class: badge_class)
+      badge_class ||= 'success'
```

Comment:
```suggestion
    def badge(message: message, badge_class: 'success')
```

Reply:
you know im sure this was intentional, like you want that second keyword arg to be explicit in the method call? which if so, totally fine.

Reply:
right right 👍

Reply:
oh this reply is sort of in the wrong thread kinda

Reply:
wow weird, how did that happen!

Reply:
> you know im sure this was intentional, like you want that second keyword arg to be explicit in the method call? which if so, totally fine.

I do not recall any intention of why I did it like that, possibly after changing a bunch that is what I left with and forgot to clean it up, your suggestions makes more sense

Reply:
really more of a code style type of review comment, not a big deal either way